lib/Sema/CSDiagnostics.h

```diff
@@ -1939,12 +1939,15 @@ class ArgumentMismatchFailure : public ContextualFailure {
   bool diagnoseMisplacedMissingArgument() const;
 };
 
-/// Replace a coercion ('as') with a forced checked cast ('as!').
-class MissingForcedDowncastFailure final : public ContextualFailure {
+/// Replace a coercion ('as') with a runtime checked cast ('as!' or 'as?').
+class MissingDowncastFailure final : public ContextualFailure {
```

Comment:
A better name could be - `InvalidCoercionFailure` since we are replacing coercion `as` with some kind of downcast (`as!` or `as?`).

Reply:
Sure! fixed :)